vault/core.go

```diff
@@ -1029,6 +1035,7 @@ func CreateCore(conf *CoreConfig) (*Core, error) {
 		pendingRemovalMountsAllowed:    conf.PendingRemovalMountsAllowed,
 		expirationRevokeRetryBase:      conf.ExpirationRevokeRetryBase,
 		rollbackMountPathMetrics:       conf.MetricSink.TelemetryConsts.RollbackMetricsIncludeMountPoint,
+		disableLeaseRoleTracking:       conf.DisableLeaseRoleTracking,
```

Comment:
You'll need to do something similar in NewTestCluster to be able to test this properly.

Reply:
thanks! is this sufficient: https://github.com/hashicorp/vault/pull/22651/files#diff-07bbf23b142e3fba2ba348837bc6814262f0cabd708bcc3fb9ff52a91ef079d3R1548